hl_gangs.sp

```diff
@@ -2798,7 +2801,14 @@ int GetPlayerAliveCount(int team)
 		}
 	}
 	return iAmmount;
- }
+}
+
+// to avoid this https://user-images.githubusercontent.com/3672466/28637962-0d324952-724c-11e7-8b27-15ff021f0a59.png
+void SanitizeName(char[] name)
+{
+	ReplaceString(name, MAX_NAME_LENGTH, "#", "?");
```

Comment:
I don't get it. How can a '#' in a nickname result in such a broken menu like linked above? Could you explain this to me?

Reply:
create a menu item with "0 #buy" in the display text and see for yourself :p